debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/Scn.java

```diff
@@ -15,29 +15,91 @@
  */
 public class Scn implements Comparable<Scn> {
 
+    /**
+     * Represents an Scn that is considered INVALID, useful as a placeholder.
+     */
     public static final Scn INVALID = new Scn(new BigDecimal(-1));
```

Comment:
Is there any chance for these to collide with actual SCN values?

Reply:
It shouldn't as the documentation says its a monotonically increasing sequence.